cluster/bin/snappy-job.sh

```diff
@@ -127,7 +127,7 @@ while (( "$#" )); do
         echo "The netrc file $passwordfile not found."
         exit 1
       fi
-      securePart=" --netrc-file ${passwordfile}"
+      securePart=" --config ${passwordfile}"
```

Comment:
Don't understand this change. Isn't config is too generic to identify password file?